src/Utils/Strings.php

```diff
@@ -332,10 +332,10 @@ public static function compare($left, $right, $len = NULL)
 	 * @param  string|array
 	 * @return string
 	 */
-	public static function findPrefix($strings)
+	public static function findPrefix(...$strings)
 	{
-		if (!is_array($strings)) {
-			$strings = func_get_args();
+		if (is_array($strings[0])) {
+			$strings = $strings[0];
```

Comment:
:+1: The possibility to typehint the value with variadics is good.

> Passing directly array is slightly faster on PHP 7.

That makes sense, but is the difference noticeable in runtime?